lib/interceptor/response-error.js

```diff
@@ -0,0 +1,112 @@
+'use strict'
+
+const { parseHeaders } = require('../core/util')
+const createHttpError = require('http-errors')
```

Comment:
```suggestion
const http = require('node:http')
```

Reply:
Thank you so much for correcting me @ronag

Reply:
i doubt that tihs is correct?